e2e/cypress/integration/api/iam/01_projects_api.spec.ts

```diff
@@ -168,12 +173,23 @@ describeIfIAMV2p1('projects API: applying project', () => {
         method: 'GET',
         url: `/apis/iam/v2beta/projects/${project.id}`
       }).then((response) => {
-        expect(response.body.project.status).to.equal('NO_RULES');
+        expect(response.body.project.status).to.equal(noRulesStr);
       });
     }
 
-    for (const rule of [avengersRule, xmenRule]) {
+    cy.request({
+      headers: { 'api-token': Cypress.env('adminTokenValue') },
+      method: 'GET',
+      url: '/apis/iam/v2beta/projects'
+    }).then((response) => {
+      response.body.projects.forEach((project: { id: string; status: any; }) => {
```

Comment:
[nit] I'd prefer a filter/forEach construct here (and below), but what we have is fine.
```typescript
response.body.projects
  .filter({ id, status } => project.id === avengersProject.id || project.id === xmenProject.id)
  .forEach({ status } => expect(project.status).to.equal(noRulesStr));
```